main.cpp

```diff
@@ -42,7 +42,108 @@ send me a DM to check your pull request
  */
 
 #include <iostream>
+
+struct FloatType
+{
+    float add(float lhs, float rhs)
+    {
+        return lhs + rhs;
+    }
+
+    float subtract(float lhs, float rhs)
+    {
+        return lhs - rhs;
+    }
+
+    float multiply(float lhs, float rhs)
+    {
+        return lhs * rhs;
+    }
+
+    float divide(float lhs, float rhs)
+    {
+        if(rhs == 0.f) // gave me warning when writing 0 without .f saif it was unsafe to use == or =! with floats. Don't see how 0.f makes it better
+        std::cout << "Attention: Dividing floats by 0!" << "\n";
+        return lhs / rhs;
```

Comment:
You'll learn how to solve this warning in a later project part.  You can ignore them for now.
- [ ] fix the indentation of the `if()` body